### DISCUSSION TITLE
The preferable way of communications about topics which is hard to fit into GH issues/discussion

### DISCUSSION THREAD

**Discussion by SemyonSinchenko** [apache/incubator-graphar#368] [General]
What is the preferable way to communicate with other GAR devs outside of GH? For example, if I want to ping a specific dev in a PR/issue, or to discuss the question which is hard to fit into issues/discussions? Or even notify other contributors about my vacation? I know there is a slack, but it looks dead (I do not like slack too). Maybe some alternative way? Mailing lists? Some chat-application?

**Comment by acezen** [comment]
Hi, Sem, we have a mailing list place in googlegroups, you can send an email to [graphar+subscribe@googlegroups.com](mailto:graphar+subscribe@googlegroups.com).

There is also a weekly community meeting for developer, you can see detail at https://github.com/alibaba/GraphAr/wiki/GraphAr-Weekly-Community-Meeting

We have try slack and discord, and they work not well, If you have other communication tool that fit for GraphAr community, feel free to suggest.

**Comment by acezen** [comment]
Hi, Sem, just got your discussion about use poetry to install all python dependencies. I think all you want to discussion is fit to github's discussion. That if you open a discussion, github would send a notification to all developer of Github, and it's easy to convert a discussion to a issue and keep the develop log of GraphAr in github.
And keep the communication way unified and concise is good for the project.
What do you think? Feel free to tell me the shortage of the issue/discussion of github
